From e94e6cf402b68cda1110af96011401d034e47a86 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Wed, 29 Jul 2020 15:44:53 +0800
Subject: [PATCH] Test case1 for Change library instance

Description: Change the file guid of library instance (TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf).

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf b/TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf
index aaf6858291..504660b3bb 100644
--- a/TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf
+++ b/TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf
@@ -15,7 +15,7 @@
   INF_VERSION                    = 0x00010005
   BASE_NAME                      = BaseFakeLibraryLib
   MODULE_UNI_FILE                = BaseFakeLibraryLib.uni
-  FILE_GUID                      = 9172fe8b-9a36-40f8-bba5-5d48e45370ec
+  FILE_GUID                      = 9172fe8b-9a34-40f8-b645-5f48e683e0ec
   MODULE_TYPE                    = BASE
   VERSION_STRING                 = 1.0
   LIBRARY_CLASS                  = FakeLibraryLib
-- 
2.27.0.windows.1

